config/routes.rb

```diff
@@ -2,7 +2,7 @@
 
   root 'art_labels#index'
 
-  resources :art_labels, only: [:index, :new, :create, :show, :destroy] do
+  resources :art_labels, only: [:index, :new, :create, :show, :edit, :update, :destroy] do
```

Comment:
I would get rid of the only now that you're using all possible actions for the resource!